taxonomy-manager-rest-server/src/test/java/com/digirati/taxman/rest/server/ServerConceptResourceTest.java

```diff
@@ -22,12 +28,42 @@ public void createConcept_JsonLd() {
     @Test
     public void createConcept_JsonLdFailsValidationNoPrefLabel() {
         // @formatter:off
-        givenJsonLdRequest(getClass(), "concept--create-no-pref-label.json")
+        givenJsonLdRequest(getClass(), "concept--create-no-pref-label.json", Collections.emptyMap())
                 .when()
                     .post("/v0.1/concept")
                 .then()
                     .statusCode(422);
         // @formatter:on
     }
 
+    @Test
+    public void createUpdate_RetainsDctermsSource() {
+        // @formatter:off
+        String conceptLocation =
+                givenJsonLdRequest(getClass(), "concept--create-with-uri.json")
+                    .when()
+                        .post("/v0.1/concept")
+                    .then()
+                        .body("'dcterms:source'.'@id'", equalTo("https://example.org/my-id-001"))
+                    .and()
+                    .extract()
+                        .header("Location");
+
+        givenJsonLdRequest(getClass(), "concept--update-with-source.json", Map.of("@id", conceptLocation))
+                .when()
+                    .put(conceptLocation)
+                .then()
+                    .statusCode(204);
```

Comment:
Technically this is updating pre-existing `dcterms:source` with a new one rather than sending the existing non-default and expecting the Concept to have unchanged `dcterms:source`...

...but this is the same thing essentially, so this is a more robust test (allowing FE to put desired value, useful for fixing broken stuff I guess.)

Reply:
On the other side of this: sending a `Concept` resource to taxman without a `dcterms:source` property is telling the backend you've updated the concept, and removed the `dcterms:source` field. `PUT`s are always full updates.

Whether or not that is what we want in all cases is still open for discussion, but the general model of idempotent updates seems to be working so far.